model/host/host.go

```diff
@@ -760,6 +760,19 @@ func FindAllContainers() ([]Host, error) {
 	return hosts, nil
 }
 
+// FindAllParents finds all the parents
```

Comment:
Nit: `FindAllParents finds all hosts that have child containers`. Remember that later readers may not know what a "parent" is, so write your comments in as human-friendly ways as you can.